src/passes/InstrumentLocals.cpp

```diff
@@ -135,45 +135,48 @@ struct InstrumentLocals : public WalkerPass<PostWalker<InstrumentLocals>> {
     Builder builder(*getModule());
     Name import;
     auto type = curr->value->type;
-    if (type.isFunction()) {
-      import = set_funcref;
-    } else {
-      TODO_SINGLE_COMPOUND(curr->value->type);
-      switch (type.getBasic()) {
-        case Type::i32:
-          import = set_i32;
-          break;
-        case Type::i64:
-          return; // TODO
-        case Type::f32:
-          import = set_f32;
-          break;
-        case Type::f64:
-          import = set_f64;
-          break;
-        case Type::v128:
-          import = set_v128;
-          break;
-        case Type::externref:
-          import = set_externref;
-          break;
-        case Type::exnref:
-          import = set_exnref;
-          break;
-        case Type::anyref:
-          import = set_anyref;
-          break;
-        case Type::eqref:
-          import = set_eqref;
-          break;
-        case Type::i31ref:
-          import = set_i31ref;
-          break;
-        case Type::unreachable:
-          return; // nothing to do here
-        default:
-          WASM_UNREACHABLE("unexpected type");
-      }
+    if (type.isFunction() && type != Type::funcref) {
+      // FIXME: support typed function references
```

Comment:
Why not continue aborting in this case since it still isn't implemented?

Reply:
I see. How about checking the features and aborting only if `typed-function-references` is enabled?

Reply:
Doing that would still break people running with `-all`, including our existing tests. I can work around it in the tests, but I'm not sure it's worth it?

Reply:
Ok, that's reasonable. I still think this isn't great, but I agree that having to change all the tests to not use `-all` is worse, so there's not anything to be done about it.